src/megaapi.cpp

```diff
@@ -2211,6 +2211,11 @@ void MegaApi::getAccountDetails(MegaRequestListener *listener)
     pImpl->getAccountDetails(true, true, true, false, false, false, listener);
 }
 
+void MegaApi::getSpecificAccountDetails(bool storage, bool transfer, bool pro, MegaRequestListener *listener)
+{
+    pImpl->getAccountDetails(storage, transfer, pro, false, false, false, listener);
+}
+
```

Comment:
I will fix it, since only Android is using that function and they don't care about the storage/transfer/pro data returned by the extended version, so it won't ask for those details.